velox/type/Conversions.h

```diff
@@ -34,16 +34,19 @@ namespace facebook::velox::util {
 struct PrestoCastPolicy {
   static constexpr bool truncate = false;
   static constexpr bool legacyCast = false;
+  static constexpr bool throwOnUnicode = true;
```

Comment:
Please add a TODO comment so that we remember to remove this flag once casting from unicode is fixed.